docs/topics/gradle/gradle-configure-project.md

```diff
@@ -50,7 +50,8 @@ In the following table, there are the minimum and maximum **fully supported** ve
 
 | KGP version   | Gradle min and max versions           | AGP min and max versions                            |
 |---------------|---------------------------------------|-----------------------------------------------------|
-| 1.9.20–1.9.23 | %minGradleVersion%–%maxGradleVersion% | %minAndroidGradleVersion%–%maxAndroidGradleVersion% |
+| 2.0.0         | %minGradleVersion%–%maxGradleVersion% | %minAndroidGradleVersion%–%maxAndroidGradleVersion% |
+| 1.9.20–1.9.23 | 6.8.3–8.7                             | 7.1.3–8.3.1                                         |
```

Comment:
Unfortunately we still have some issues for 8.6 and 8.7 Gradle releases in Kotlin 2.0.0. So while we are running tests with these releases, fully supported release is only Gradle 8.5.

Reply:
Ok, thanks. Fixed!